src/sonic-yang-mgmt/setup.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```

Comment:
Are you going to merge this into src/sonic-yang-models

Reply:
sonic-yang-models will have its own separate setup.py